code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -5,6 +5,7 @@
 	item_state = "beacon"
 	var/code = "electronic"
 	origin_tech = "bluespace=1"
+	dog_fashion = null

```

Comment:
Dogs can wear station radios, but they can't wear any of their children objects. In this particular case, inheritance is a curse, but short of manually specifying a list of types to allow (which ruins the whole point), will just have to override children with no sprites manually. The alternative is allow children, but that will mean the sprite is wrong.